### PR TITLE
setup kristen TESLA bucket

### PR DESCRIPTION
**Pull request by xdoan**
setting up Kristen's TESLA bucket, added my info to template.

### CODE REVIEW

File: config/prod/kristen-bucket.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: "remote/s3-bucket-v2.yaml"
```
Comment (zaro0508) [inline config/prod/kristen-bucket.yaml:3]: should be "remote/s3-bucket-v2.j2"

File: config/prod/kristen-bucket.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  # LifecycleDataStorageClass: 'INTELLIGENT_TIERING'
+  # (Optional) Number of days until S3 objects are moved to the LifecycleDataStorageClass, default is 30
+  # LifecycleDataTransition: '90'
+  # (Optional) Number of days (from creation) when objects are deleted from S3 and LifecycleDataStorageClass, default is 365000
+  # LifecycleDataExpiration: '1825'
+
+# For CI system (do not change)
+hooks:
+  before_launch:
+    - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.us-east-1.amazonaws.com/aws-infra/master/s3-bucket.yaml --create-dirs -o templates/remote/s3-bucket.yaml"
```
Comment (zaro0508) [inline config/prod/kristen-bucket.yaml:48]: should be `... /master/s3-bucket-v2.j2 --create-dirs -o templates/remote/s3-bucket-v2.j2`

File: config/prod/kristen-bucket.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: "remote/s3-bucket-v2.yaml"
+stack_name: "kristen-bucket"
```
Comment (zaro0508) [inline config/prod/kristen-bucket.yaml:4]: bucket name containing `kristen-bucket` is ok?  wouldn't you rather this bucket's name be something related to the project?

**Review by zaro0508 — CHANGES_REQUESTED** [review]
please fix the build failure

File: config/prod/kristen-bucket.yaml (hunk 3 above)
Comment (xdoan) [reply config/prod/kristen-bucket.yaml:4]: ah I wasn't sure if Kristen wanted that name or not. Do you want me to change the template name if I change the stack name?

Comment (zaro0508) [reply config/prod/kristen-bucket.yaml:4]: yes, stack_name and file name must match.

**Comment by xdoan** [comment]
fixed build failure (twice) thank you for the help!

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**